src/main/java/APIWorker.java

```diff
@@ -0,0 +1,50 @@
+import com.google.gson.*;
```

Comment:
I think it's better to explicitly list the classes that you use instead of wildcard imports